input/metadata/end_to_end_testing/open_lot_e2e_metadata.json

```
@@ -28,12 +28,12 @@
       "type": "json",
       "properties": "animal_population_properties"
     },
-    "animal_net_merit": {
-      "title": "Animal Net Merit data",
-      "description": "The net merit value for cows.",
-      "path": "input/data/animal/animal_genetics/NetMerit_HO.csv",
+    "animal_mean_phenotype": {
+      "title": "Average animal phenotype data by birth year",
+      "description": "The average animal phenotype data by birth year.",
+      "path": "input/data/animal_genetics/average_phenotype.csv",
```

```suggestion
      "title": "Mean animal phenotype data by birth year",
      "description": "The mean animal phenotype data by birth year.",
      "path": "input/data/animal_genetics/mean_phenotype.csv",
```

Should either be mean or average consistently imo.